src/bindings/python/tests/test_runtime/test_tensor.py

```diff
@@ -148,6 +149,62 @@ def test_init_with_numpy_copy_memory(ov_type, numpy_dtype):
     assert ov_tensor.byte_size == arr.nbytes
 
 
+def test_init_with_node_output_port():
+    param1 = ops.parameter(ov.Shape([1, 3, 2, 2]), dtype=np.float64)
+    param2 = ops.parameter(ov.Shape([1, 3, 32, 32]), dtype=np.float64)
+    param3 = ops.parameter(ov.PartialShape.dynamic(), dtype=np.float64)
+    ones_arr = np.ones(shape=(1, 3, 32, 32), dtype=np.float64)
+    tensor1 = Tensor(param1.output(0))
+    tensor2 = Tensor(param2.output(0), ones_arr)
+    tensor3 = Tensor(param3.output(0))
+    with pytest.warns(RuntimeWarning):
+        tensor4 = Tensor(param3.output(0), ones_arr)
```

Comment:
Should dynamic shapes be thrown as an warning? If the shape is dynamic, it does not really matter what is an array's shape. Can the check be adjusted to not throw warning when the shape is dynamic?

Reply:
updated